molecule/default/tests/test_tomcat.yml

```diff
@@ -0,0 +1,64 @@
+---
+
+service:
+  tomcat:
+    enabled: true
+    running: true
+
+user:
+  tomcat:
+    exists: true
+    groups:
+      - tomcat
+
+group:
+  tomcat:
+    exists: true
+
+port:
+  tcp:8080:
+    listening: true
+  tcp:8005:
```

Comment:
Maybe the AJP port should be tested too :)

Reply:
done